Yosemite/Yosemite/Model/WooPlans/LegacyWooPlan.swift

```diff
@@ -136,7 +136,7 @@ public struct LegacyWooPlan: Decodable {
     }
 }
 
-public enum AvailableInAppPurchasesWPComPlans: String {
+public enum AvailableInAppPurchasesWPComPlans: String, CaseIterable {
```

Comment:
We can remove the `CaseIterable` conformance, I just realized I left if there from my first attempt to tackle this. Will do before merging.

```suggestion
public enum AvailableInAppPurchasesWPComPlans: String {
```